core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-share-access/ngbd-modal-workflow-share-access.component.ts

```diff
@@ -44,6 +49,11 @@ export class NgbdModalWorkflowShareAccessComponent implements OnInit {
     this.refreshGrantedList(workflow);
   }
 
+  public onChange(value: string): void {
+    this.filteredOwners = [];
+    this.filteredOwners = this.allOwners.filter(owner => owner.toLowerCase().indexOf(value.toLowerCase()) !== -1);
```

Comment:
I get "`TypeError: Cannot read properties of undefined (reading 'toLowerCase')`" error whenever I open the share workflow panel because `value` is `undefined`. You need to fix this bug.